index.html

```diff
@@ -320,7 +320,7 @@ <h2 id="civname">Loading…</h2>
             if (civsConfig[`${civ}`]) {
                 // Fast-Fail horses if civ disablesHorses
                 if (civsConfig[`${civ}`]['disableHorses'] &&
-                    horseDisabled.includes(name)) {
+                    horseDisabledBuildings.concat(horseDisabledUnits).concat(horseDisabledTech).includes(name)) {
```

Comment:
Probably not a big performance issue, but it feels inefficient to repeatedly concat these arrays inside of the `for` loop